pkg/services/services_test.go

```diff
@@ -20,15 +22,27 @@ type ServicesTestSuite struct {
 }
 
 func TestServicesTestSuite(t *testing.T) {
-	db, err := sqlx.Connect("postgres", "user=postgres sslmode=disable")
-	fmt.Println(err)
+	config := testhelpers.NewConfig()
 	logger := zap.NewNop()
-	store := storage.NewStore(db, logger)
+
+	dbConfig := storage.DBConfig{
+		Host:     config.GetString(appconfig.DBHostConfigKey),
+		Port:     config.GetString(appconfig.DBPortConfigKey),
+		Database: config.GetString(appconfig.DBNameConfigKey),
+		Username: config.GetString(appconfig.DBUsernameConfigKey),
+		Password: config.GetString(appconfig.DBPasswordConfigKey),
```

Comment:
For tests, do we also need to set the env to local? Or is that already happening? Otherwise, we'll fail on it being an empty string, right?

Reply:
Yeah, let me add a test env and put it in the `easi test` command.